x/campaign/types/message_edit_campaign.go

```diff
@@ -45,5 +47,12 @@ func (msg *MsgUpdateCampaignName) ValidateBasic() error {
 	if err := CheckCampaignName(msg.Name); err != nil {
 		return sdkerrors.Wrap(ErrInvalidCampaignName, err.Error())
 	}
+
+	// TODO parameterize
+	if len(msg.Metadata) > spntypes.MaxMetadataLength {
```

Comment:
We should allows providing empty campaign name for not updating it
Since I believe `CheckCampaignName` fail for empty name, `CheckCampaignName` is only checked if the name is non-null
The message ValidateBasic should fail if none of the two value are provided